hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationStats.java

```diff
@@ -208,11 +208,10 @@ public String formatToString(boolean detailed) {
             s.append(", elapsedMigrationOperationTime=").append(getElapsedMigrationOperationTime()).append("ms")
                     .append(", totalElapsedMigrationOperationTime=").append(getTotalElapsedMigrationOperationTime()).append("ms")
                     .append(", elapsedDestinationCommitTime=").append(getElapsedDestinationCommitTime()).append("ms")
-                    .append(", totalElapsedDestinationCommitTime=").append(getTotalElapsedDestinationCommitTime()).append("ms");
+                    .append(", totalElapsedDestinationCommitTime=").append(getTotalElapsedDestinationCommitTime()).append("ms")
+                    .append(", elapsedMigrationTime=").append(getElapsedMigrationTime()).append("ms")
```

Comment:
These numbers show the total elapsed time of the individual migrations. But when migrations are concurrent, they might be misleading. Because they don't show the absolute elapsed time between start and complete. Still I can revert this change, if you think it's useful...

Reply:
oh right, so with concurrent migrations it will be more like "cpu time" rather than wall clock time. Let's leave this only in `detailed == true` case since it can be confusing.